tools/run_python_system_tests.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import pip
+import time
+import json
+import zipfile
+import argparse
+import tempfile
+import urllib.request
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d', '--driver', default='niswitch', type=str)
+args = parser.parse_args()
+
+
+# Install Tox
```

Comment:
Since you have a print, the comment is no longer necessary. This applies elsewhere.

Reply:
Fixed